### DISCUSSION TITLE
API Rest  I need help

### DISCUSSION THREAD

**Discussion by tocasoft** [Budibase/budibase#6063] [Help]
Hi
I followed the steps below to execute an API query but I have no result, I'm new to this, please if you could help me what I'm doing wrong.
 thank you very much

![Budibase](https://user-images.githubusercontent.com/2348489/170292200-42a9db7b-2e5e-41fb-a88e-0f3b126d4f72.png)
![Budibase2](https://user-images.githubusercontent.com/2348489/170292220-5eb1c613-3a77-4168-85be-c9b82648117d.png)
![Budibase3](https://user-images.githubusercontent.com/2348489/170292229-4f2109f5-e210-476f-80bc-cdf9abbd6185.png)
![Budibase4](https://user-images.githubusercontent.com/2348489/170292243-055b61f3-ec07-4acd-9836-a804403df6a6.png)

**Comment by mike12345567** [comment]
Hi @tocasoft - I've converted your issue to a discussion as it is a request for assistance rather than a bug or feature request.

It looks like the format of your URL may not be correct and I'm not sure if you've included your API key in the headers, please review our API documentation: https://docs.budibase.com/reference

**Reply by tocasoft** [reply]
Thanks for your reply. I followed the instructions according to https://github.com/Budibase/budibase/issues/2511 and I was guided by this documentation https://docs.budibase.com/docs/rest  that you recommend or you have a video that can help me. 
Many thanks

**Comment by melohagan** [comment]
Hi @tocasoft 

You are trying to hit an endpoint `<mydomain>.budibase.app/api/test1`

I don't think such an endpoint exists, e.g. the endpoint is Not Found.

**Reply by tocasoft** [reply]
Thank for you time.
I don't know what endpoint I can use . How I can create this endpoint. Do you have something reference that help me please.
Thanks

**Reply by mike12345567** [reply]
Hi @tocasoft - we have documentation for the REST API as part of our docs, I posted a link in my original comment: https://docs.budibase.com/reference

This documents all the API endpoints which are available, which you can use. You can't create new endpoints, you can only utilise the ones that already exist.

**Reply by tocasoft** [reply]
Thanks for you response. Let me check it. thanks
